.github/workflows/token_refresh.yml

```diff
@@ -1,14 +0,0 @@
----
-name: refresh_automation_hub_token
```

Comment:
only stays in netcommon, can be reused from there

Reply:
I am of the opinion that this should live in ansible.utils and not in ansible.netcommon since netcommon depends on utils and utils is usually the first collection to be released whenever we do releases.